dist/config.js

```diff
@@ -3,7 +3,7 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
-const API = exports.API = Object.freeze({
+var API = exports.API = Object.freeze({
```

Comment:
Why are these changing?

Reply:
the babel config i used targets older browsers that dont support `const`